python/vyos/ifconfig/bond.py

```diff
@@ -433,6 +458,10 @@ def update(self, config):
             if mode == '802.3ad':
                 self.set_lacp_rate(config.get('lacp_rate'))
 
+            # Add system mac address for 802.3ad
+            if mode == '802.3ad' and 'system_mac' in config:
+                self.set_system_mac(config.get('system_mac'))
```

Comment:
What happen‘s on switching mode? The system_mac persists? Or is the sysfs entry removed by the Kernel?

Reply:
if it's switched is not used in the other modes ,so , it's removed the sysfs entry by the kerne: 

```
vyos@bond0# compare
[interfaces bonding bond0]
- mode "802.3ad"
+ mode "active-backup"

[edit]
vyos@bond0# commit
[edit]
vyos@bond0# sudo cat /proc/net/bonding/bond0
Ethernet Channel Bonding Driver: v6.6.28-amd64-vyos

Bonding Mode: fault-tolerance (active-backup)
Primary Slave: None
Currently Active Slave: eth1
MII Status: up
MII Polling Interval (ms): 100
Up Delay (ms): 0
Down Delay (ms): 0
Peer Notification Delay (ms): 0

Slave Interface: eth1
MII Status: up
Speed: 1000 Mbps
Duplex: full
Link Failure Count: 0
Permanent HW addr: 50:00:00:04:00:01
Slave queue ID: 0
```

Reply:
If mode is 802.3ad and the `system-mac` CLI node is removed, the previously configured `system-mac` remains programmed to the Kernel. What was the previous default? Local MAC address?

Reply:
```console
cpo@BR1.wueI:~$ sudo hexdump /sys/class/net/bond10/bonding/ad_actor_system
0000000 3030 303a 3a30 3030 303a 3a30 3030 303a
0000010 0a30
0000012
```
```console
cpo@BR1.wueI:~$ sudo cat /sys/class/net/bond10/bonding/ad_actor_system
00:00:00:00:00:00
```

So the default is now clear and should be restored when the CLI node is removed.

Reply:
Will be added by a commit of mine